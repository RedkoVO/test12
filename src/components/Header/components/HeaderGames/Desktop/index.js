import React from 'react'
import PropTypes from 'prop-types'
import withStyles from '@material-ui/core/styles/withStyles'

import UserImg from '../../../../../assets/images/user.png'

import styles from './styles'

const HeaderCatalog = ({ classes, children }) => (
  <div className={classes.root}>
    <div className={classes.catalogUserBlock}>
      <p className={classes.catalogUserHello}> Hello, Ben A <br />
        <span className={classes.catalogUserVerification}>KYC VERIFICATED</span>
      </p>
      <div className={classes.catalogUserPhoto}>
        <img src={UserImg} alt="" />
      </div>
      <p className={classes.catalogUserBallance}> 764.23 DCB <br />
        <span className={classes.catalogUserBallDescr}>YOUR BALANCE</span>
      </p>
    </div>
    
    {children}
  </div>
)

HeaderCatalog.propTypes = {
  classes: PropTypes.object,
  children: PropTypes.node
}

export default withStyles(styles)(HeaderCatalog)