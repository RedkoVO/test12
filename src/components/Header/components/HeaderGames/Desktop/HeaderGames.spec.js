import * as React from 'react'
import renderer from 'react-test-renderer'

import HeaderGames from './'

it('renders correctly DESKTOP', () => {
  const tree = renderer.create(<HeaderGames isDesktop={true} children={<div>test</div>} />).toJSON()
  expect(tree).toMatchSnapshot()
})

it('renders correctly MOBILE', () => {
  const tree = renderer.create(<HeaderGames isDesktop={false} children={<div>test</div>} />).toJSON()
  expect(tree).toMatchSnapshot()
})