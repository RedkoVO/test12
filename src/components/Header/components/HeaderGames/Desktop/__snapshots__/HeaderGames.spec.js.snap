// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly DESKTOP 1`] = `
<div
  className="HeaderCatalog-root-1"
>
  <div
    className="HeaderCatalog-catalogUserBlock-2"
  >
    <p
      className="HeaderCatalog-catalogUserHello-3"
    >
       Hello, Ben A 
      <br />
      <span
        className="HeaderCatalog-catalogUserVerification-5"
      >
        KYC VERIFICATED
      </span>
    </p>
    <div
      className="HeaderCatalog-catalogUserPhoto-7"
    >
      <img
        alt=""
        src="user.png"
      />
    </div>
    <p
      className="HeaderCatalog-catalogUserBallance-4"
    >
       764.23 DCB 
      <br />
      <span
        className="HeaderCatalog-catalogUserBallDescr-6"
      >
        YOUR BALANCE
      </span>
    </p>
  </div>
  <div>
    test
  </div>
</div>
`;

exports[`renders correctly MOBILE 1`] = `
<div
  className="HeaderCatalog-root-1"
>
  <div
    className="HeaderCatalog-catalogUserBlock-2"
  >
    <p
      className="HeaderCatalog-catalogUserHello-3"
    >
       Hello, Ben A 
      <br />
      <span
        className="HeaderCatalog-catalogUserVerification-5"
      >
        KYC VERIFICATED
      </span>
    </p>
    <div
      className="HeaderCatalog-catalogUserPhoto-7"
    >
      <img
        alt=""
        src="user.png"
      />
    </div>
    <p
      className="HeaderCatalog-catalogUserBallance-4"
    >
       764.23 DCB 
      <br />
      <span
        className="HeaderCatalog-catalogUserBallDescr-6"
      >
        YOUR BALANCE
      </span>
    </p>
  </div>
  <div>
    test
  </div>
</div>
`;
